rosetta-ide/pom.xml

```diff
@@ -45,6 +45,15 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>${project.basedir}/src-gen/main/java</directory>
+                <includes>
+                    <include>META-INF/services/org.eclipse.xtext.ISetup</include>
+                </includes>
+            </resource>
+        </resources>
```

Comment:
Fix for issue David and James were experiencing on the BSP since the Tycho removal.